web/ui/src/Services/ServiceDetailsController.js

```diff
@@ -644,15 +669,70 @@
             }
         };
 
```

Comment:
Its bad practice to touch the DOM in an angular app, so getting rid of these DOM based view updates and using regular data binding is good news.